packages/features/shell/Shell.tsx

```diff
@@ -539,9 +539,7 @@ const navigation: NavigationItemType[] = [
     isCurrent: ({ router, item }) => {
       const path = router.asPath.split("?")[0];
       // During Server rendering path is /v2/apps but on client it becomes /apps(weird..)
-      return (
-        (path.startsWith(item.href) || path.startsWith("/v2" + item.href)) && !path.includes("routing-forms/")
```

Comment:
We would be accessing routing-forms as /routing-forms now instead of /apps/routing-forms